presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/CommandExecutor.java

```diff
@@ -13,25 +13,41 @@
  */
 package com.teradata.prestomanager.agent;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
 
 import static java.lang.Runtime.getRuntime;
 import static java.lang.String.format;
 
-// TODO: Add logger
 public final class CommandExecutor
 {
+    private static final Logger LOGGER = LogManager.getLogger(CommandExecutor.class);
+
     private CommandExecutor() {}
 
     public static int executeCommand(String command)
             throws PrestoManagerException
     {
-        try {
+        LOGGER.debug("Command executed: {}", command);
+
+        try  {
             Process process = getRuntime().exec(command);
+            Stream<String> inputStream = new BufferedReader(new InputStreamReader(process.getInputStream())).lines();
+            Stream<String> errorStream = new BufferedReader(new InputStreamReader(process.getErrorStream())).lines();
+            inputStream.forEach(line -> LOGGER.debug("{}", line));
```

Comment:
I strongly recommend against this type of logging; the subprocess' output should not have separate log entries for each line, it should be logged all together, and probably in a different file from most of the Presto Manager logs.

And it should be logged at the `info` level, I think.